src/Avalonia.Themes.Fluent/Controls/AutoCompleteBox.xaml

```diff
@@ -3,7 +3,7 @@
   <Design.PreviewWith>
     <Border Padding="20">
       <AutoCompleteBox Width="200">
-        <AutoCompleteBox.Items>
+        <AutoCompleteBox.ItemsSource>
```

Comment:
Uh... how does this even work? Afaics `AutoCompleteBox.ItemsSource` isn't initialized with an empty collection so I'd have though this would result in an NRE?

Reply:
Yes, it doesn't. Rider showed an error and I automatically replaced Items with ItemsSource without double checking.